scripts/smtp_collector.py

```diff
@@ -93,7 +93,7 @@ def process_message(self, peer, mailfrom, rcpttos, data):
 
                 def _getfname(counter):
                     return current_test_folder\
-                        .join("%s-%d.eml" % (arrived.strftime("%Y%m%d%H%M%S"), int(counter)))
+                        .join("{}-{}.eml".format(arrived.strftime("%Y%m%d%H%M%S"), int(counter)))
```

Comment:
datetime format strings can be inline

if counter is casted to int, i wonder if a :d format works